contents/blog/open-source-telemetry-ethical.md

```diff
@@ -0,0 +1,98 @@
+---
+date: 2020-07-30
+title: The ethics of open source product analytics
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+---
+
+A topic that strikes fear into the hearts of many working on open source projects - telemetry. So much so, I was a little nervous about writing this piece!
+
+## A framework for considering what is ethical
+
+### Duty-based ethics
```

Comment:
I think you need to put a few lines of text above this before you hit them with heavy duty terms. "There are frameworks to think about how ethics work, i'll just talk about a few" or something; make it a bit more casual

Reply:
Echoing Tim here - I'd still like some words in between "An ethics framework" and "Duty-based ethics"